react/PasswordExample/styles.styl

```diff
@@ -1,5 +1,5 @@
 .Token--number
-    color var(--dodgerBlue)
+    color #297EF2 // dodgerBlue
```

Comment:
Should we import the palette and use it as `palette.dodgerBlue` instead of having an  other absolute colors? We already previously defined the absolute color for dodgerBlue, no? (hard to remember the start of this PR ^^')

Reply:
Yep, done, don't know what I was thinking here ;)